wazo_dxtora/__init__.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/python3
+# Copyright 2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from .dxtora import main  # noqa
```

Comment:
Don't understand why you need to import in this file?
The shebang is probably useless
instead of noqa, we should use 
```python
__all__ = [
    'main',
]
```